src/database/entity/Role.ts

```diff
@@ -0,0 +1,17 @@
+import { Column, Entity, OneToMany, type Relation } from 'typeorm';
+import { CommonEntity } from '@/database/utils/CommonEntity';
+import { User } from './User';
+
+@Entity()
+export class Role extends CommonEntity {
+	@Column({
+		type: 'varchar',
+		length: 30,
+		nullable: false,
+		unique: true,
+	})
+	name: string;
```

Comment:
idealně enum aby jsi na to mohl pohodlně odkazovat

udělej si v konstantách
const roles = ["user","admin"] as const

potom si udělej 
type Role = (typeof roles)[number]

v enumu budeš mít ten roles a v typu budeš mít ten Role